build.gradle

```diff
@@ -20,7 +20,12 @@ subprojects {
 	repositories {
 		mavenCentral()
 
-		maven { url = 'https://repo.spongepowered.org/repository/maven-public/' }
+		maven {
+			url = 'https://repo.spongepowered.org/repository/maven-public/'
```

Comment:
Just wondering, why is this exclude necessary?

Reply:
It was pulling Fabric API from spongepowered maven, it was a debugging step in fixing FAPI deprecated stuff..